app/javascript/modules/listingDetailsAside/ListingDetailsSeeTheUnit.tsx

```diff
@@ -28,6 +29,39 @@ const SeeTheUnitSubsection = ({ title, children, headingClass }: SubsectionProps
   )
 }
 
+const SeeDetailsOnline = (listing: RailsListing) => {
+  if (!listing.Multiple_Listing_Service_URL && !listing.Listing_Online_Details) {
+    return null
+  }
+
+  return (
+    <SeeTheUnitSubsection title={t("seeTheUnit.seeDetailsOnline")}>
+      <div className="flex-row space-y-1">
+        {listing.Multiple_Listing_Service_URL && (
+          <div>
+            <Link
+              className="no-underline"
+              href={listing.Multiple_Listing_Service_URL}
+              hideExternalLinkIcon={true}
+            >
+              {t("listings.process.seeTheUnitOnMls")}
+            </Link>
+          </div>
+        )}
+        {listing.Listing_Online_Details?.map((detail: ListingOnlineDetail) => {
+          return (
+            <div key={detail.Id}>
+              <Link className="no-underline" href={detail.URL} hideExternalLinkIcon={true}>
+                {detail.Name}
```

Comment:
Is there another ticket to implement the new machine translations functionality here?

Reply:
Yes we have https://sfgovdt.jira.com/browse/DAH-2623